### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by AFLLAMA**

### CODE REVIEW

File: lesson_02/bin/check_jobs.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+JOB2_PORT = 8082
+
+RAW_DIR = os.path.join(BASE_DIR, "lesson_02", "data", "raw", "sales", "2022-08-09")
+STG_DIR = os.path.join(BASE_DIR, "lesson_02", "data", "stg", "sales", "2022-08-09")
+
+
+def run_job1():
+    print("Starting job1:")
+    resp = requests.post(
+        url=f"http://localhost:{JOB1_PORT}/",
```
Comment (tmikhailovska) [inline lesson_02/bin/check_jobs.py]: no need to hard code the host value, host and port value should be added as function parameters

File: lesson_02/bin/check_jobs.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        json={"date": "2022-08-09", "raw_dir": RAW_DIR},
+    )
+    assert resp.status_code == 201
+    print("job1 completed!")
+
+
+def run_job2():
+    print("Starting job2:")
+    resp = requests.post(
+        url=f"http://localhost:{JOB2_PORT}/",
```
Comment (tmikhailovska) [inline lesson_02/bin/check_jobs.py]: the same

File: lesson_02/api/main.py
Comment (tmikhailovska) [inline lesson_02/api/main.py:1]: It is not clear why this file is needed

File: lesson_02/data/raw/sales/2022-08-09/2022-08-09_2.json
Comment (tmikhailovska) [inline lesson_02/data/raw/sales/2022-08-09/2022-08-09_2.json:1]: the data not need to store in git

File: lesson_02/bin/check_jobs.py (hunk 1 above)
Comment (AFLLAMA) [reply lesson_02/bin/check_jobs.py]: fixed

File: lesson_02/bin/check_jobs.py (hunk 2 above)
Comment (AFLLAMA) [reply lesson_02/bin/check_jobs.py]: fixed

File: lesson_02/api/main.py
Comment (AFLLAMA) [reply lesson_02/api/main.py:1]: I was testing the connection to the server. 
Removed from the repo.

File: lesson_02/data/raw/sales/2022-08-09/2022-08-09_2.json
Comment (AFLLAMA) [reply lesson_02/data/raw/sales/2022-08-09/2022-08-09_2.json:1]: Removed it from git.
Can I expect a mark correction of 10 points?

**Review by tmikhailovska — APPROVED** [review]

**Pull request merged**